Makefile

```diff
@@ -0,0 +1,26 @@
+NODE_BIN=./node_modules/.bin
+
+help:
+	@echo '                                                                                     		'
+	@echo 'Makefile for the xqueue-watcher                                                   		'
+	@echo '                                                                                     		'
+	@echo 'Usage:                                                                               		'
+	@echo '    make requirements                 install requirements for local development     		'
+	@echo '    make test                         install requirements for local development     		'
+	@echo '    make clean                        delete generated byte code and coverage reports		'
+	@echo '                                                                                     		'
+
+requirements: 
+	pip install -qr requirements/production.txt --exists-action w
```

Comment:
should this be -q -r ? If -qr works, we should make the next `pip` statement under test.requirements consistent.